static/app/views/admin/adminSettings.tsx

```diff
@@ -105,6 +109,13 @@ export default class AdminSettings extends AsyncView<{}, State> {
               {fields['performance.issues.n_plus_one_db.count_threshold']}
               {fields['performance.issues.n_plus_one_db.duration_threshold']}
             </Panel>
+            <Panel>
+              <PanelHeader>Performance Issues - Consecutive DB Detector</PanelHeader>
+              {fields['performance.issues.consecutive_db.problem-creation']}
+              {fields['performance.issues.consecutive_db.la-rollout']}
+              {fields['performance.issues.consecutive_db.ea-rollout']}
+              {fields['performance.issues.consecutive_db.ga-rollout']}
+            </Panel>
             <Panel>
               <PanelHeader>Performance Issues - N+1 API Calls Detector</PanelHeader>
               {fields['performance.issues.n_plus_one_api_calls.problem-creation']}
```

Comment:
Should we gate these options behind their respective feature flags?

Reply:
Actually nevermind, I just remembered these options are meant to be available only to us